<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewGenerixDerivation3Page</class>
 <widget class="QWizardPage" name="NewGenerixDerivation3Page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Define option for derivation project</string>
  </property>
  <property name="subTitle">
   <string>In this page you can define a prefix to use in your derivation path and the possibility to copy the original file in the destination path too.</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="m_missingDirChk">
     <property name="text">
      <string>&amp;Create automatically missing directory</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_addPrefixChk">
     <property name="text">
      <string>&amp;Prefix derivated files</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>17</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="m_prefixEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="m_copyToChk">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>C&amp;opy source file in the derivation path</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_addPrefixChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_copyToChk</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_addPrefixChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_prefixEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
